### PR TITLE
[WIP] Add missing 'setBreakpointCondition' action to Breakpoint props

### LINKED ISSUE #7272: [breakpoints] "remove condition", "enable" & "disable" context menu items not working
STR
  Set a breakpoint and add a condition to it. 
  In the breakpoints list, right click on the breakpoint and select "remove condition"

ER
  breakpoint condition is removed

AR
  breakpoint condition is not removed

### PR DESCRIPTION
**Pull request by kenjiO**
Fixes #7272 

In BreakpointContextMenu the `setBreakpointCondition` prop is undefined.  I added it to `mapStateToProps` in Breakpoint which passes it to BreakpointContextMenu.  I also added a mochi test.

### CODE REVIEW

**Comment by kenjiO** [comment]
There is a similar issue with the `toggleDisabledBreakpoint` action.  I'll create a fix and test for that one too.

File: src/test/mochitest/browser_dbg-breakpoints-cond.js [hunk 1]
```diff
@@ -81,4 +81,12 @@ add_task(async function() {
   bp = findBreakpoint(dbg, "simple2", 5);
   is(bp.condition, "1", "breakpoint is created with the condition");
   assertEditorBreakpoint(dbg, 5, true);
+
+  //right click breakpoint in breakpoints list
+  rightClickElement(dbg, "breakpointItem", 3)
+  // select "remove condition";
+  selectMenuItem(dbg, 9);
+  await waitForDispatch(dbg, "SET_BREAKPOINT_CONDITION");
```
Comment (jasonLaster) [inline src/test/mochitest/browser_dbg-breakpoints-cond.js]: this is probably safe, but i'm paranoid with timing issues

```suggestion
  const bpCondition = waitForDispatch(dbg, "SET_BREAKPOINT_CONDITION");
  selectMenuItem(dbg, 9);
  await bpCondition;
```

**Comment by jasonLaster** [comment]
hmm, i'm now seeing an issue 
browser_dbg-breakpoints-cond.js

**Comment by kenjiO** [comment]
I tried the github "suggested change" beta feature that automatically turned your suggestion into a commit.  But the suggestion diff didn't delete the existing `selectMenuItem(dbg, 9);` line. So now that line is there twice.  It passes on my mac but failed on travis.  I'm trying to see if removing the duplicate line fixes it now.

**Comment by kenjiO** [comment]
@jasonLaster  I don't know why the Travis test is failing with the change of creating the waitForDispatch promise before the mouse events.  It passes when I run the test on my mac.  Do you have any ideas?

**Comment by jasonLaster** [comment]
Thanks for the hard work ironing out the test. Sorry for the timing headache :/

**Pull request merged**